tests/test_rlp.py

```diff
@@ -0,0 +1,412 @@
+import json
+import os
+from typing import List, Sequence, Tuple, Union, cast
+
+import pytest
+
+from eth1spec import rlp
+from eth1spec.eth_types import U256, Bytes, Uint
+from eth1spec.rlp import RLP
+
+from .helpers import hex2bytes
+
+#
+# Tests for RLP encode
+#
+
+#
+# Testing bytes encoding
+#
+
+
+def test_rlp_encode_empty_bytes() -> None:
+    assert rlp.encode_bytes(b"") == bytearray([128])
```

Comment:
Is there a reason to use a decimal version in these tests vs hex? I would rather these be hex values since you don't need to also do the computation in your head.